docs/sources/administration/correlations/use-correlations-in-visualizations/index.md

```diff
@@ -3,7 +3,7 @@ labels:
   products:
     - enterprise
     - oss
-title: Use correlations in visualizations
+title: 'Example: Use correlations in visualizations'
```

Comment:
I don't think this change should be made as it changes the information architecture of the section.

I believe this is a "task topic" and should conform to the advice in https://grafana.com/docs/writers-toolkit/structure/topic-types/task/

```suggestion
title: Use correlations in visualizations
```